cmd/otelcontribcol/go.mod

```diff
@@ -207,7 +207,6 @@ require (
 	go.opentelemetry.io/collector/consumer v0.93.1-0.20240125183026-3cacd40b27e8
 	go.opentelemetry.io/collector/exporter v0.93.1-0.20240125183026-3cacd40b27e8
 	go.opentelemetry.io/collector/exporter/debugexporter v0.93.1-0.20240125183026-3cacd40b27e8
```

Comment:
I assume debugexporter should use the same version everywhere

Reply:
hmmm... i ran the script locally.... and now i just realized i missed committing a couple of files... *sigh*

Reply:
should be fixed now